web/themes/custom/pineapple/sass/components/navigation/_breadcrumb.scss

```diff
@@ -0,0 +1,22 @@
+.breadcrumb {
+  font-size: $small-font;
+  color: $dark-grey;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  padding-bottom: 0;
+
+  a {
+    color: $dark-grey;
+
+    &:hover,
+    &:focus {
+      color: $black;
+      text-decoration: underline;
+    }
+  }
+}
+
+
+hr {
+  border-bottom: 1px solid $light-grey;
+}
```

Comment:
This styling makes not difference. and the hr should be lighter to match the designs

Reply:
done